ui/apps/platform/src/services/ComplianceIntegrationService.ts

```diff
@@ -0,0 +1,25 @@
+import axios from 'services/instance';
+
+import { complianceV2Url } from './ComplianceCommon';
+
+const complianceIntegrationsBaseUrl = `${complianceV2Url}/integrations`;
+
+export interface ComplianceIntegration {
```

Comment:
UI almost always declares data structures as `type` instead of `interface`

Reply:
updated here and one in `ComplianceResultsService`